vision:
  surface_frame_name: id_0

initialization:
  initial_pose_offset: [0.0, 0.0, 0.15] # w.r.t the surface_frame_name
  initial_approach_angle: 0.62831853071 # in radians
  manipulation_action_name: /gazebo/dexterous_manipulation
  approach_action_name: /gazebo/approach_controller
  move_action_name: /gazebo/move_controller
  gravity_compensation_service_name: /ft/compensated

experiment:
  tool_frame_name: l_gripper_tool_frame
  base_link_name: torso_lift_link
  num_of_experiments: 1
  use_vision: false
  sim_mode: true
